pgcli/pgexecute.py

```diff
@@ -24,6 +26,10 @@
 # See http://initd.org/psycopg/articles/2014/07/20/cancelling-postgresql-statements-python/
 ext.set_wait_callback(psycopg2.extras.wait_select)
 
+FunctionMetadata = namedtuple('FunctionMetadata',
+                              ['schema_name', 'func_name', 'arg_list', 'result',
```

Comment:
What is the `result` attribute here? Is it the return_type? If so, should we call it `return_type`?